test_runner/src/test/kotlin/ftl/ios/xctest/common/FindTestsForTargetKtTest.kt

```diff
@@ -0,0 +1,88 @@
+package ftl.ios.xctest.common
+
+import com.google.common.truth.Truth.assertThat
+import ftl.config.FtlConstants
+import ftl.ios.xctest.FIXTURES_PATH
+import ftl.ios.xctest.multiTargetsSwiftXcTestRunV1
+import ftl.ios.xctest.swiftTestsV1
+import ftl.ios.xctest.swiftXcTestRunV1
+import ftl.run.exception.FlankGeneralError
+import org.junit.Assume.assumeFalse
+import org.junit.Test
+import java.io.File
+import java.nio.file.Files
+import java.nio.file.Paths
+
+class FindTestsForTargetKtTest {
+
+    @Test
+    fun `findTestNames respects skip`() {
+        assumeFalse(FtlConstants.isWindows)
```

Comment:
we have this here, but what about the other tests?

Reply:
FYI, most of the tests in this PR are moved from
* https://github.com/Flank/flank/blob/ab55fbcb02f6cfe99b3b502db6bf52f661cd34ca/test_runner/src/test/kotlin/ftl/ios/XctestrunTest.kt and
* https://github.com/Flank/flank/blob/ab55fbcb02f6cfe99b3b502db6bf52f661cd34ca/test_runner/src/test/kotlin/ftl/ios/ParseTest.kt

to other files to match the refactor of associated functions.


The new tests which I have added are

* https://github.com/Flank/flank/blob/4c3b55d30723aefc9ca65b1bec4415a095217531/test_runner/src/test/kotlin/ftl/ios/xctest/FindXcTestNamesV2KtTest.kt
* https://github.com/Flank/flank/blob/4c3b55d30723aefc9ca65b1bec4415a095217531/test_runner/src/test/kotlin/ftl/ios/xctest/RewriteXcTestRunV2KtTest.kt

I have no windows to test it so I have followed convention with `assumeFalse(FtlConstants.isWindows)`

Reply:
makes sense. We will need a fixup task for windows again.

Reply:
> We will need a fixup task for windows again.

"Again" in this context sounds like that was working previously, then was broken, fixed and now is broken again. But It's not true for this test.

As I said, nothing changes here except file name.